File: kotlin-back/src/main/kotlin/com/example/kotlinback/member/dto/MemberType.java
package com.example.kotlinback.member.dto;

import lombok.Getter;

@Getter
public enum MemberType {
    KAKAO("K", "kakao"),
    NAVER("N", "naver"),
    GOOGLE("G", "google"),
    COMMON("C", "common");

    private String code;
    private String typeName;

    MemberType(String code, String typeName) {
        this.code = code;
        this.typeName = typeName;
    }
}

File: kotlin-back/src/main/kotlin/com/example/kotlinback/member/dto/MemberType.kt
package com.example.kotlinback.member.dto


enum class MemberType(private val code: String, private val typeName: String) {
    KAKAO("K", "kakao"),
    NAVER("N", "naver"),
    GOOGLE("G", "google"),
    COMMON("C", "common")
}
